### ISSUE TITLE
Is it possible to use Perple_X to compute H20 and CO2 EOS? 

### ISSUE THREAD

**Issue by Francyrad** [jadconnolly/Perple_X#6]
Is it possible? When I usually compute a material i can only choose % of oxides. I ask then If there is an escamotage to do that

Francesco

**Comment by jadconnolly** [comment]
Yes, you can evaluate CO2 and H2O EoS directly using FLUIDS, FRENDLY, MEEMUM or VERTEX. The latter three programs require that you specify a thermodynamic data file.

**Comment by Francyrad** [comment]
Is there some tutorial how to do that? The purpose is to use them in ASPECT

**Comment by jadconnolly** [comment]
FLUIDS is, I hope, intuitive. FRENDLY requires: that you select a thermodynamic data file (e.g., hp02ver.dat), that you add the line "GFSM T" to perplex_option.dat, and that you select the phase of interest ("H2O" or "CO2"). The calculation with MEEMUM or VERTEX is as that of any other phase diagram section, as with FRENDLY I recommend adding "GFSM T" to the option file. If you have no experience running Perple_X one of the tutorials at www.perplex.ethz.ch may be helpful.

**Comment by Francyrad** [comment]
Thank you, i'll check them out!
